build.gradle

```diff
@@ -17,7 +17,7 @@ buildscript {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath "com.google.protobuf:protobuf-gradle-plugin:$protobufVersion"
         classpath "androidx.navigation:navigation-safe-args-gradle-plugin:$navVersion"
-        classpath "org.jlleitschuh.gradle:ktlint-gradle:9.2.1"
+        classpath "org.jlleitschuh.gradle:ktlint-gradle:9.4.1"
```

Comment:
Why does this change here?

Reply:
My ktlint build was failing and it was complaining about some trailing commas which kotlin 1.4 now supports, so I thought of updating the ktlint plugin.

Reply:
If it still passes on this PR I'm okay with it.